### ISSUE TITLE
Remove/reduce minimum 10 character _key value constraint from all graph objects

### ISSUE THREAD

**Issue by austinkelleher** [JupiterOne/data-model#81]
Currently, there is a constraint in the base [`GraphObject` schema](https://github.com/JupiterOne/data-model/blob/master/src/schemas/GraphObject.json#L13) that all graph object `_key` values must be at minimum 10 characters. This has lead to some integration developers adding artificial prefixes at the beginning of `_key` values just to avoid it. 

We should consider reducing the constraint to a single character or at least reducing the constraint lower than 10 characters.

cc @aiwilliams @ndowmon @mknoedel

**Comment by aiwilliams** [comment]
I agree, that constraint should be removed. We should be able to count on the the combination of `_type` and `_key` (scope, really) to prevent duplicates, as the key tracker does for us.

**Comment by mknoedel** [comment]
💯 👍 🥇
